cli/src/maintenance.rs

```diff
@@ -742,6 +768,39 @@ impl SolidoState {
         }
     }
 
+    pub fn try_send_anker_rewards(&self) -> Option<MaintenanceInstruction> {
+        let anker_state = self.anker_state.as_ref()?;
+
+        if anker_state.ust_reserve_balance == MicroUst(0) {
+            return None;
+        }
+
+        // Use the low 32 bits of the current slot as the nonce. This ensures
+        // that we don't repeat the nonce for at least 2^32 slots, which is 327
+        // years even if the block times were 100ms, but in practice block times
+        // are 600-700ms, so it’s even longer. Do we never send multiple
+        // transactions based on the same slot though? If we do, then the state
+        // we observe should also be identical, and only one of the transactions
+        // should succeed.
+        let wormhole_nonce = (self.clock.slot & 0xffff_ffff) as u32;
+
+        let (instruction, additional_signer) = anker_state.get_send_rewards_instruction(
+            self.solido_address,
+            self.maintainer_address,
+            wormhole_nonce,
+        );
+
+        let ust_amount = anker_state.ust_reserve_balance;
+        let output = MaintenanceOutput::SendRewards { ust_amount };
+        println!("Trying to send rewards: {:?}", &output);
```

Comment:
Is this a debug msg? Cause it can mess the tests (if we have one)

Reply:
Yes, good catch, that is a debugging leftover, removed now. We shouldn’t have bare `println!`s in the CLI code because it will mess up the `--output json` mode.